packages/stash/README.md

```diff
@@ -0,0 +1,79 @@
+# Stash
+
+Stash is a client state library optimized for the MUD data model.
+It uses the MUD store config to define local tables, which support writing, reading and subscribing to table updates.
+It comes with a query engine optimized for "ECS-style" queries (similar to `@latticexyz/recs`) but with native support for composite keys.
```

Comment:
is there a docs page we can link to to clarify "ECS-style queries"?

Reply:
added a link to the ECS explainer